packages/notebook-extension/src/index.ts

```diff
@@ -2073,6 +2087,14 @@ function populateMenus(
     }
   } as IRunMenu.ICodeRunner<NotebookPanel>);
 
+  // Add a renderAllMarkdown group to the run menu.
+  const renderAllMarkdown = [
+    CommandIDs.renderAllMarkdown,
+    CommandIDs.run,
```

Comment:
Can you remove `run` and `runInConsole` here? The first is redundant with existing menu items, and the second should probably have its own discussion. I think it's fine for `renderAllMarkdown` to have its own menu group.